waterlog/src/Components/CostsReport.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import DailyCostsReports from './DailyCostsReports';
+import MonthlyCostsReports from './MonthlyCostsReports';
+import { fetchCostsDaily, fetchCostsMonthly, fetchCostsSeasonally } from '../actions/CostsReportActions';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import SeasonalCostReports from './SeasonalCostReports';
+
+class CostsReport extends Component {
+    constructor(props) {
+        super(props);
+        this.openGraph = this.openGraph.bind(this);
+        this.state = {
+            display: "daily"
+        }
+    }
+    componentDidMount() {
+        this.openGraph("daily");
+        this.props.fetchCostsDaily();
+        this.props.fetchCostsMonthly();
+        this.props.fetchCostsSeasonally();
+    }
+    openGraph = (graphType) => {
+        this.setState({
+            display: graphType
+        })
+    }
+    getGraphType = () => {
+        if (this.state.display === "daily")
```

Comment:
Wouldn't it be better to have a case statement with consts for daily, monthly or seasonal?

Reply:
Thank you, I will consider this.

Reply:
ok